ironfish-cli/src/commands/wallet/chainport/status.ts

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import axios from 'axios'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+import { getSupportedNetworkDetails } from '../../../utils/chainport'
+import { watchTransaction } from '../../../utils/transaction'
+
+type ChainportTransactionStatus = {
+  base_network_id?: number
+  base_tx_hash?: string
+  base_tx_status?: number
+  base_token_address?: string
+  target_network_id?: number
+  target_tx_hash?: string
+  target_tx_status?: number
+  target_token_address?: string
+  created_at?: string
+  port_in_ack?: boolean
+}
+
+export class StatusCommand extends IronfishCommand {
+  static description = `Display an account transaction`
+  static hidden = true
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'hash',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      required: true,
+      description: 'Hash of the transaction',
+    },
+    {
+      name: 'account',
+      required: false,
+      description: 'Name of the account',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { args } = await this.parse(StatusCommand)
+    const hash = args.hash as string
+    const client = await this.sdk.connectRpc()
+    const account = args.account as string | undefined
+
+    this.logger.log('Transaction status on Ironfish:')
+
+    await watchTransaction({
```

Comment:
Instead of this being follow by default, I think it should output non-blocking, but have a `-f` option for following.